pyproject.toml

```
@@ -3,42 +3,51 @@ requires = ["setuptools>=61.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "afcharts_py"
+name = "afcharts"
 version = "0.1.0"
-description = "A python package to produce charts following UK Government Analysis Function Guidance"
+description = "Standardised plotting configuration tools to meet Analysis Function plotting guidelines."
```

I prefer the previous version
```suggestion
description = "A python package to produce charts following UK Government Analysis Function guidance"
```